MediaBrowser.Controller/Net/BasePeriodicWebSocketListener.cs

```
@@ -41,6 +50,8 @@ protected BasePeriodicWebSocketListener(ILogger<BasePeriodicWebSocketListener<TR
             ArgumentNullException.ThrowIfNull(logger);
 
             Logger = logger;
+
+            _ = HandleMessages();
```

Discarding the task without explicit error handling in `HandleMessages` is an antipattern and should be avoided

---

There's a try catch?

---

If `_channel.Reader.ReadAllAsync()` throws an exception it gets swallowed